core/client/scenes/scene-world.js

```diff
@@ -36,6 +36,11 @@ WorldScene = new Phaser.Class({
     Phaser.Scene.call(this, { key: 'WorldScene' });
   },
 
+  preload() {
+    const url = 'https://raw.githubusercontent.com/rexrainbow/phaser3-rex-notes/master/dist/rexpinchplugin.min.js';
```

Comment:
I wonder if we should not host this file directly in the project. We don't have control over the version and the state of the server. In `client/plugin` for example?

Reply:
Wouldn't it be better in the BootScene? The WorldScene is reloaded each time you change levels

Reply:
All done 🙌

Reply:
Thank you!